app/uk/gov/hmrc/agentoverseasfrontend/views/application/main_business_address.scala.html

```diff
@@ -18,140 +18,83 @@
-                  <span class="visuallyhidden">@Messages("error.prefix")</span>
-                  @Messages(formError.message)
-              </span>
-              }
-
-              <input
-              type="text"
-              id="addressLine3"
-              class="form-control--block"
-              name="addressLine3"
-              spellcheck="false"
-              autocomplete="address-level2"
-                  @mainBusinessAddressForm.data.get("addressLine3").map { v => value="@v"}
-                  @if(mainBusinessAddressForm.error("addressLine3").isDefined){aria-describedby="addressLine3-error"}
-              >
-
-          </div>
-
-          <div class="form-group">
-              <label for="addressLine4">@Messages("mainBusinessAddress.form.addressLine4")</label>
-              @mainBusinessAddressForm.error("addressLine4").map { formError => <span class="error-notification" id="addressLine4-error">
-                  <span class="visuallyhidden">@Messages("error.prefix")</span>
-                  @Messages(formError.message)
-              </span>
-              }
-
-              <input
-              type="text"
-              id="addressLine4"
-              class="form-control--block"
-              name="addressLine4"
-              spellcheck="false"
-              autocomplete="postal-code"
-                  @mainBusinessAddressForm.data.get("addressLine4").map { v => value="@v"}
-                  @if(mainBusinessAddressForm.error("addressLine4").isDefined){aria-describedby="addressLine4-error"}
-              >
-
-          </div>
-
-          <div class="form-group @if(mainBusinessAddressForm.error("countryCode").isDefined){form-field--error soft--ends}">
-              <label for="countryCode-auto-complete">@Messages("mainBusinessAddress.form.country")</label>
-              @mainBusinessAddressForm.error("countryCode").map { formError => <span class="error-notification" id="countryCode-error">
-                  <span class="visuallyhidden">@Messages("error.prefix")</span>
-                  @Messages(formError.message)
-              </span>
-              }
-              <div class="autocomplete-wrapper" id="countryCode">
-                  <select id="countryCode-auto-complete" name="countryCode">
-                      <option value=""></option>
-                      @for(option <- countries.toSeq) {
-                          <option
-                          id="countryCode-@option._1"
-                          value="@option._1"
-                              @if(value==option._1){selected="selected"}>@option._2</option>
-                      }
-                  </select>
-              </div>
-          </div>
-      </fieldset>
+    @form(action = routes.TradingAddressController.submitMainBusinessAddress, 'class -> "form js-form") {
 
-      <button class="button margin-bottom-30" type="submit" id="continue">@Messages("button.continue")</button>
+      @govukInput(Input(
```

Comment:
on multi-input forms, all the inputs should be wrapped in a fieldset and the heading should be wrapped in a legend.

```
<fieldset>
<legend>
<h1></h1>
</legend>
// inputs //
</fieldset>
```
The Date, Radio and Checkbox components provide them by default but multiple text inputs we have to add them manually, either with the Fieldset component or via html with new classes